node/src/chain_spec.rs

```diff
@@ -589,6 +589,63 @@ pub fn calamari_testnet_config(id: ParaId) -> CalamariChainSpec {
 	)
 }
 
+#[cfg(feature = "calamari")]
+pub fn calamari_testnet_config_ci(id: ParaId) -> CalamariChainSpec {
+	let properties = calamari_properties();
+
+	// (controller_account, aura_id)
+	let initial_authorities: Vec<(AccountId, AuraId)> = vec![
+		(
+			hex!["d664b6e69e3b1bcd89274bf90a2abf409baa9a4cd9072194cdc70ead1ba65f0c"].into(),
+			hex!["ee73f78b7dd29f30902c1a3bd1e4a6fcc2f26be088343d3ee011e2660fd02a66"]
+				.unchecked_into(),
+		),
+		(
+			hex!["b4cc4dcffaa95be696f76b8b68ea114f46edc67f343f6aacabe8e5000af38b50"].into(),
+			hex!["7cd4af9ad51d443740f71ecd5850385e98985224628c5ea08209bb2015523f3c"]
+				.unchecked_into(),
+		),
+		(
+			hex!["c68bbab6a85bd17548f61ab26a52c577c12748f08062cffe921316a7283fbc00"].into(),
+			hex!["b40aa6bd104d0260b60350c2fb30d4882437466d66135130b667799ea6c9f52b"]
+				.unchecked_into(),
+		),
+		(
+			hex!["20e3ddcf0c6c456f2b9d035d3309fe84f43c2c5abc524a7fc903ed3d813c714b"].into(),
+			hex!["4a3aa51469e802be6504422cd9dd03be638ac3f6dc3a7c0c85a6ace3e72f0048"]
+				.unchecked_into(),
+		),
+		(
+			hex!["a2057091a6d15a9d64c398f148395b69aaae99626cffa53ac5e6beaa5790f946"].into(),
+			hex!["a68feb4fe2ea3f8ff288af4254aad2284e1cd0da67cb9ea61c13632bad57eb40"]
+				.unchecked_into(),
+		),
+	];
+
+	let root_key: AccountId =
+		hex!["04ac7fe1675e10c0df9c136f53df912e6ae7d2c21778a293c534226f8a3e6a0c"].into();
```

Comment:
I split this into its own method because we don't have a solution in place to handle the corresponding secrets yet.
Generated new ones and directly use them for CI.